test/native/setup.js

```diff
@@ -81,6 +81,7 @@ jest.mock( '@wordpress/api-fetch', () => {
 jest.mock( '@wordpress/react-native-bridge', () => {
 	return {
 		addEventListener: jest.fn(),
+		logException: jest.fn(),
```

Comment:
Good catch! Since this function is only invoked when there's an error, we overlooked adding it because unit tests were passing. Thanks @dcalhoun for fixing it 🙇 !